src/signals/IncidentMap/components/FilterPanel/FilterPanel.test.tsx

```diff
@@ -97,6 +96,29 @@ describe('FilterPanel', () => {
       })
   })
 
+  it('should set toggle a main category without subcategories', () => {
+    const fetchResponseWithFilters = {
+      ...useFetchResponse,
+      data: fetchCategoriesResponse,
+    }
+
+    jest.mocked(useFetch).mockImplementation(() => fetchResponseWithFilters)
+
+    renderFilterPanel({ filters: mockFiltersLong })
+
+    const testCategory = 'Openbaar groen en water'
+
+    const checkbox = screen.getByTestId(testCategory)
+
+    userEvent.click(checkbox)
+
+    expect(
+      mockSetFilters.mock.calls[1][0].find(
+        (filter: Filter) => filter.name === testCategory
+      ).filterActive
+    ).toBe(false)
+  })
+
```

Comment:
Maybe good to check whether the filterActive was true initially.

Reply:
yeah, well its true by default, but true, can add it to assure.